handbook/data.md

```diff
@@ -275,6 +275,45 @@ The following is the description of the air pollution exposure data.
 ```
 `````
 
+### PM2.5 US High Resolution Grid, 2000-2016
+
+`````{dropdown} PM2.5 US Grid
+```{list-table}
+:header-rows: 0
+
+* - spatial_coverage
+  - US
+* - spatial_resolution
+  - 1km x 1km
+* - temporal_coverage
+  - 2000-2016
+* - temporal_resolution
+  - annually
+* - size
+  - ~80 MB/year
+* - fasse_location
+  - `/n/dominici_nsaph_l3/Lab/data/exposure/pm25/whole_us/annual/grid_pts/qd_new_predictions`.
+* - processing_description
+  - Merge by row the 1-column matrix PM2.5 values (`PredictionStep2_Annual_PM25_USGrid_20**0101_20**1231.rds`) with the corresponding 1km x 1km United States Grid Matrix (`USGridSite.rds`).
+* - data_visualization
```

Comment:
I also suggest not using one-off metadata fields. We can have a list of standard metadata fields that people should use. That will make it easier to export data descriptions to other catalogs if need be etc.

Reply:
I.e., we didn't have `data_visualization` before. This could be `git_repository` or added as a sentence in the `description`